docs/api-markdown-documenter/alert-node.js

```diff
@@ -44,20 +41,4 @@ class AlertNode extends DocumentationParentNodeBase {
 		this.alertKind = alertKind;
 		this.title = title;
 	}
-
-	/**
-	 * Generates an `AlertNode` from the provided string.
-	 *
-	 * @param {string} text - The node's text content.
-	 * @param {string} alertKind - The kind of alert.
-	 * @param {string | undefined} title - (Optional) Title text for the alert.
-	 */
-	static createFromPlainText(text, alertKind, title) {
```

Comment:
Not used, it seems?

Reply:
Apparently not, and the code it referenced was removed from `api-markdown-documenter` a while back.